source/ota.c

```diff
@@ -1232,7 +1240,7 @@ static DocParseErr_t extractAndStoreArray( char * pKey,
             err = DocParseErrOutOfMemory;
 
             LogError( ( "Memory allocation failed "
-                        "[key: valueLength]=[%s: %s]",
+                        "[key: valueLength]=[%s: %lu]",
                         pKey,
                         valueLength ) );
```

Comment:
I think valueLength here should be cast to a long.  We recently fixed logging in the rest of the repos, such that values are cast to standard types, and int32_t is cast to long.  These changes should be made wherever format strings are used.  See https://github.com/FreeRTOS/coreMQTT/pull/113

Of course, it might be better to do such in a separate PR.

Reply:
I will make a note of this and raise another PR.